openfermioncirq/experiments/hfvqe/objective.py

```diff
@@ -0,0 +1,173 @@
+from typing import Optional
+from itertools import product
+import numpy as np
+from openfermion.ops import InteractionOperator, InteractionRDM
+from openfermion.utils import wedge
+from openfermion.transforms import get_fermion_operator
+from openfermioncirq.experiments.hfvqe.circuits import rhf_params_to_matrix
+# pylint: disable=C
+
+
+def get_matrix_of_eigs(w: np.ndarray) -> np.ndarray:
+    """
```

Comment:
Use `r"""` instead of `"""` when putting latex inside, to avoid bad escape sequences.